### PR TITLE
 switch to flit (and one package)

### LINKED ISSUE #217: switch to flit (and one package)
> Currently it's impossible to distribute multiple modules/packages with flit.

I'd like to move everything under the `modernize` name and use `modernize/__main__.py`

### PR DESCRIPTION
**Pull request by graingert**
fixes #217

### CODE REVIEW

**Comment by graingert** [comment]
@takluyver

File: modernize/__main__.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 from __future__ import generator_stop
```
Comment (graingert) [inline modernize/__main__.py:1]: @brettcannon I'm not really sure what this file was doing outside of a package. It didn't seem to work, so I moved it in here where it does

File: modernize/utils.py [hunk 2]
```diff
@@ -5,10 +5,8 @@
 from fissix.pygram import python_symbols as syms
```
Comment (graingert) [inline modernize/utils.py:5]: this was renamed to boht avoid cyclic imports and defs in `__init__.py`

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -40,3 +40,15 @@ repos:
     hooks:
       - id: prettier
         args: [--prose-wrap=always, --print-width=88]
+
+  - repo: https://github.com/myint/autoflake
+    rev: v1.4
+    hooks:
+      - id: autoflake
+        exclude: &fixtures tests/functional/|tests/input|tests/extensions/data|tests/regrtest_data/|tests/data/
```
Comment (graingert) [inline .pre-commit-config.yaml:48]:
```suggestion
```

**Review by jreese — APPROVED** [review]

**Pull request merged**